VCFv4.3.tex

```diff
@@ -577,43 +591,113 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 For precise variants, the consensus sequence the alternate allele assembly is derivable from the REF and ALT fields.
 However, the alternate allele assembly file may contain additional information about the characteristics of the alt allele contigs.
 
 \footnotesize
 \begin{verbatim}
-##INFO=<ID=MEINFO,Number=4,Type=String,Description="Mobile element info of the form NAME,START,END,POLARITY">
-##INFO=<ID=METRANS,Number=4,Type=String,Description="Mobile element transduction info of the form CHR,START,END,POLARITY">
-##INFO=<ID=DGVID,Number=1,Type=String,Description="ID of this element in Database of Genomic Variation">
-##INFO=<ID=DBVARID,Number=1,Type=String,Description="ID of this element in DBVAR">
-##INFO=<ID=DBRIPID,Number=1,Type=String,Description="ID of this element in DBRIP">
-##INFO=<ID=MATEID,Number=.,Type=String,Description="ID of mate breakends">
-##INFO=<ID=PARID,Number=1,Type=String,Description="ID of partner breakend">
-##INFO=<ID=EVENT,Number=1,Type=String,Description="ID of event associated to breakend">
-##INFO=<ID=CILEN,Number=2,Type=Integer,Description="Confidence interval around the inserted material between breakends">
-##INFO=<ID=DP,Number=1,Type=Integer,Description="Read Depth of segment containing breakend">
-##INFO=<ID=DPADJ,Number=.,Type=Integer,Description="Read Depth of adjacency">
-##INFO=<ID=CN,Number=1,Type=Integer,Description="Copy number of segment containing breakend">
-##INFO=<ID=CNADJ,Number=.,Type=Integer,Description="Copy number of adjacency">
-##INFO=<ID=CICN,Number=2,Type=Integer,Description="Confidence interval around copy number for the segment">
+##INFO=<ID=MEINFO,Number=.,Type=String,Description="Mobile element info of the form NAME,START,END,POLARITY">
+\end{verbatim}
+\normalsize
+
+If present, the number of entries must be four (4) the number of alt allele (Number=4A).
+$MEINFO$ consists of successive quadruplets of records for each alt allele.
+
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=METRANS,Number=.,Type=String,Description="Mobile element transduction info of the form CHR,START,END,POLARITY">
+\end{verbatim}
+\normalsize
+
+If present, the number of entries must be four (4) the number of alt allele (Number=4A).
+$MEINFO$ consists of successive quadruplets of records for each alt allele.
+
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=DGVID,Number=A,Type=String,Description="ID of this element in Database of Genomic Variation">
+##INFO=<ID=DBVARID,Number=A,Type=String,Description="ID of this element in DBVAR">
+##INFO=<ID=DBRIPID,Number=A,Type=String,Description="ID of this element in DBRIP">
+##INFO=<ID=MATEID,Number=A,Type=String,Description="ID of mate breakends">
+##INFO=<ID=PARID,Number=A,Type=String,Description="ID of partner breakend">
+##INFO=<ID=EVENT,Number=A,Type=String,Description="ID of event associated to breakend">
+##INFO=<ID=CILEN,Number=.,Type=Integer,Description="Confidence interval around the inserted material between breakends">
```

Comment:
I know that you didn't change this Description text as part of this PR, but I find it really difficult to parse, and it doesn't really seem to match the explanation you've written below. Would this be a good time to rewrite it?